mimesis/builtins/ru.py

```diff
@@ -25,7 +24,7 @@ def generate_sentence(self):
         ]
         return '{0} {1} {2} {3}'.format(*sentence)
 
-    def patronymic(self, gender=None):
+    def patronymic(self, gender: str=None) -> str:
```

Comment:
Actually not only `str`. I have created type `Geneder`. Please, use it. See at example in branch `type-hinting`.